src/typegen.ts

```diff
@@ -37,8 +37,9 @@ export function doGenerate(
 }
 
 export function render(dmmf: DMMF.DMMF, photonPath: string) {
+  // photonPath contains \\ in windows. We'll replace it with a / if it occurs.
   return `\
-import * as photon from '${photonPath}';
+import * as photon from '${photonPath.replace(/\\g/, '/')}';
```

Comment:
Hm, that probably does work - but we should probably try to fix the actual photonPath to be correct in the first place, right?

Reply:
that's probably a good idea but I had no idea where it came from ^^ so any pointers would be great ^^

Reply:
I am pretty sure @Weakky or @jasonkuhrt can give us a pointer where we should look <3

Reply:
This value comes from what users pass into their plugin config. And there's a default if nothing is passed: https://github.com/prisma-labs/nexus-prisma/blob/61d6efa56301d7d4419ef9e0747447c0801b7698/src/builder.ts#L99.

I can see how the use of hardcoded `/` might not be portable. However this PR converts from `\` to `/` anyways, so I'm not fully understanding.

@wardpeet could you explain the issue you're facing a bit more?

Reply:
Definitely, when running `prisma2 generate` on windows. The `src/generated/nexus` file is being generated. 

On windows, I get this content:
```
/**
 * This file was automatically generated by GraphQL Nexus
 * Do not make changes to this file directly
 */

import * as photon from "@generated\photon"



declare global {
  interface NexusGenCustomOutputProperties<TypeName extends string> {
    crud: NexusPrisma<TypeName, 'crud'>
    model: NexusPrisma<TypeName, 'model'>
  }
}
```

I'm not sure how `@generated\photon` is passed into prisma?

Reply:
My last comment was off topic b/c was I was talking about the wrong thing.

For photon path the default is here https://github.com/prisma/nexus-prisma/blob/8c0164876bba33c459fa044119487fc50331cb52/src/builder.ts#L92 and again its something that the user can pass to the nexus-prisma plugin config.

@wardpeet IIUC you need `@generated\photon` to be `@generated/photon` in the typegen?

Reply:
I need `@generated/photon` as \ is an escape char so it won't resolve the `@generated/photon` typings.

I'm not overriding the config.input.photon also it looks like `prisma2 generate` is the culprit as it creates the wrong import inside `@generated/nexus-prisma/index.d.ts`
![image](https://user-images.githubusercontent.com/1120926/65632925-239f8700-dfdb-11e9-8a14-effbea9ebe1f.png)

Reply:
Also the next version of nexus-prisma is not a prisma generator, so this issue can be considered already fixed. A release just has to be made. I will therefore close this PR. Release will be made today or tomorrow.

Reply:
ok sounds good 👌